packages/web-components/fast-components/src/radio/radio.ts

```diff
@@ -91,6 +91,9 @@ export class Radio extends FormAssociated<HTMLInputElement> {
 
     public connectedCallback(): void {
         super.connectedCallback();
+        if (this.parentElement?.tagName !== "FAST-RADIO-GROUP") {
+            this.setAttribute("tabindex", "0");
+        }
```

Comment:
What happens if I want to set tabindex myself as well in my own implementation? Should we check for the attr before setting it here?

Reply:
yep will fix